kayenta-sql/src/main/java/com/netflix/kayenta/sql/storage/model/SqlMetricSetPairs.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2023 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.kayenta.sql.storage.model;
+
+import javax.persistence.Table;
+
+@Table(name = "metric_set_pairs")
```

Comment:
Nit: table names should typically be singular as they are describing the contents of a row within the table. You're already following that naming convention in the rest of your tables/entities.

Reply:
Yes, each row in the table will contain `metric_set_pairs`. Plural form in the given case is  `metric_set_pairs_items`